src/pages/groupComparison/Survival.tsx

```diff
@@ -317,6 +319,7 @@ export default class Survival extends React.Component<ISurvivalProps, {}> {
                             style={{ paddingBottom: 0 }}
                         >
                             {getStatisticalCautionInfo()}
+                            {getHazardRatioCautionInfo()}
```

Comment:
maybe do the same for getStatisticalCautionInfo while we're at it. (Function component)

Reply:
Also changed it for `getStatisticalCautionInfo` and also updated `/groupComparison/ClinicalData.tsx` since it also calls `getStatisticalCautionInfo`